src/dbnode/client/host_queue.go

```diff
@@ -876,7 +886,6 @@ func (q *queue) asyncFetchTagged(op *fetchTaggedOp) {
 			return
 		}
 
-		ctx, _ := thrift.NewContext(q.opts.FetchRequestTimeout())
```

Comment:
hmm, is this unused now? And we're purely relying a deadline existing on the context itself?

Reply:
Correct although it's still used for APIs that aren't performed by reverse index (simple Fetch calls).

Forcing deadline is ideal since it makes callers consider the timeout for the queries they're issuing.